azkaban-web-server/src/main/java/azkaban/imagemgmt/daos/ImageTypeDao.java

```diff
@@ -0,0 +1,9 @@
+package azkaban.imagemgmt.daos;
+
+import azkaban.imagemgmt.models.ImageType;
+import java.util.Optional;
+
+public interface ImageTypeDao {
+  public int createImageType(ImageType imageType);
+  public Optional<ImageType> getImageTypeByType(String type);
```

Comment:
Can you please rename method to getImageTypeByName?

Reply:
Done